timed/settings.py

```diff
@@ -207,34 +207,34 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 
 OIDC_DEFAULT_BASE_URL = "http://timed.local/auth/realms/timed/protocol/openid-connect"
 
+# not needed in timed-backend
+OIDC_OP_TOKEN_ENDPOINT = f"{OIDC_DEFAULT_BASE_URL}/token"
+
 OIDC_OP_USER_ENDPOINT = env.str(
-    "OIDC_USERINFO_ENDPOINT", default=default(f"{OIDC_DEFAULT_BASE_URL}/userinfo")
+    "DJANGO_OIDC_USERINFO_ENDPOINT",
+    default=default(f"{OIDC_DEFAULT_BASE_URL}/userinfo"),
 )
-OIDC_OP_TOKEN_ENDPOINT = env.str(
-    "OIDC_TOKEN_ENDPOINT", default=default(f"{OIDC_DEFAULT_BASE_URL}/token")
+OIDC_VERIFY_SSL = env.bool("DJANGO_OIDC_VERIFY_SSL", default=default(False, True))
+OIDC_CREATE_USER = env.bool("DJANGO_OIDC_CREATE_USER", default=False)
+
+OIDC_USERNAME_CLAIM = env.str(
+    "DJANGO_OIDC_USERNAME_CLAIM", default="preferred_username"
```

Comment:
Usually we use `sub` as default, but it can be configured in the client. If I understand it right, in Keycloak, `sub` is the default for access tokens, while `preferred_username` is the default for id tokens.

Reply:
changed

Reply:
Sorry, I saw it after approving it. This has not been addressed.

Reply:
Oops forgot to push it.

Reply:
This needs also to be changed in the tests.